manifest.yml

```diff
@@ -1,12 +1,12 @@
 applications:
-  - name: timesheet-codejam-<participantId>
+  - name: cloud-sdk-team-calendar-db-hdi-container
```

Comment:
The application should not use the same name as the HDI container. What was wrong with the old name? You may want to remove the notion of codejam and use something like sample instead.